app/services/mixtape_generator.rb

```diff
@@ -0,0 +1,20 @@
+class MixtapeGenerator
+  def initialize(genre:)
+    @genre = genre
+  end
+
+  def self.perform(genre:)
+    new(genre: genre).perform
+  end
+
+  def perform
+    Mixtape.new(genre: @genre, songs: pick_songs)
```

Comment:
Does the `Mixtape` need a `genre` attribute?

Isn't in going to be the unique genre on the song?

A song as you've defined it can only belong to one Genre.